ods_ci/tests/Resources/Common.robot

```diff
@@ -93,6 +93,21 @@ CSS Property Value Should Be
         Run Keyword And Continue On Failure   Should Be Equal    ${actual_value}    ${exp_value}
     END
 
+Page Should Contain A String In List
+    [Documentation]    Verifies that page contains at least one of the strings in text_list
+    [Arguments]  ${text_list}
+    FOR    ${text}    IN    @{text_list}
+        ${text_found}=    Run Keyword And Return Status    Page Should Contain   ${text}
+        IF  ${text_found}    RETURN
+    END
+    Fail    Current page doesn't contain any of the strings in: @{text_list}
+
+Wait Until Page Contains A String In List
+    [Documentation]    Waits until page contains at least one of the strings in text_list
+    [Arguments]    ${text_list}    ${retry}=12x    ${retry_interval}=5s
```

Comment:
I prefer increasing the retry and decreasing the interval to 1 second. This operation won't harm the cpu

Reply:
Now is retrying every 5s. I think it's fine like that. We don't want to stress the cluster :-)